src/uct/ib/mlx5/ib_mlx5_dv.c

```diff
@@ -303,11 +313,8 @@ static ucs_status_t uct_ib_mlx5dv_md_open(struct ibv_device *ibv_device,
         dev->pci_cswap_arg_sizes = UCT_IB_MLX5DV_GET(atomic_caps, cap, compare_swap_pci_atomic) << 2;
     }
 
-    if (has_dc) {
-        status = uct_ib_mlx5_check_dc(dev);
-    }
-
     dev->flags |= UCT_IB_DEVICE_FLAG_MLX5_PRM;
+    dev->flags |= UCT_IB_DEVICE_FLAG_DEVX;
```

Comment:
do we need to check some cap flag to see if devx is supported?

Reply:
Kernel did it when we opened device with MLX5DV_CONTEXT_FLAGS_DEVX